Tests/ProjectSpecTests/SpecLoadingTests.swift

```diff
@@ -376,9 +376,9 @@ class SpecLoadingTests: XCTestCase {
             $0.it("parses target dependencies") {
                 var targetDictionary = validTarget
                 targetDictionary["dependencies"] = [
-                    ["target": "name", "embed": false],
-                    ["target": "project/name", "embed": false],
-                    ["carthage": "name", "findFrameworks": true],
+                    ["target": "name", "embed": false, "platform": "all"],
+                    ["target": "project/name", "embed": false, "platform": "mac"],
```

Comment:
If we change to macOS
```suggestion
                    ["target": "project/name", "embed": false, "platform": "macOS"],
```

Reply:
Updated in dfc471d8511f37738bfb9a84915d822bcc299f1e.